angular.json

```diff
@@ -31,6 +31,8 @@
               "src/assets"
             ],
             "styles": [
+              "@fortawesome/fontawesome-free/css/all.min.css",
+              "bulma/css/bulma.min.css",
```

Comment:
suggestion: Nice Bulma :)) try to add it as a dependency, so other people who take your project can also use it `npm install bulma`

Reply:
Nvm I saw you add it (had some vpn issues at work with the instalation)